test/e2e/e2e_test.go

```diff
@@ -46,6 +46,9 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/autoscaling"
 	_ "k8s.io/kubernetes/test/e2e/cloud"
 	_ "k8s.io/kubernetes/test/e2e/common"
+	_ "k8s.io/kubernetes/test/e2e/common/network"
+	_ "k8s.io/kubernetes/test/e2e/common/node"
+	_ "k8s.io/kubernetes/test/e2e/common/storage"
```

Comment:
Add `test/e2e/common/imports.go` that does this, and drop these imports

This way you don't need to make the same change in `e2e_node` (it no longer sees tests in the subpackages)